theia-php-docker/next.package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia PHP Example",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/callhierarchy": "next",
+        "@theia/editor-preview": "next",
+        "@theia/file-search": "next",
+        "@theia/getting-started": "next",
+        "@theia/git": "next",
+        "@theia/json": "next",
+        "@theia/markers": "next",
+        "@theia/merge-conflicts": "next",
+        "@theia/messages": "next",
+        "@theia/mini-browser": "next",
+        "@theia/navigator": "next",
+        "@theia/outline-view": "next",
+        "@theia/plugin-ext-vscode": "next",
+        "@theia/preferences": "next",
+        "@theia/preview": "next",
+        "@theia/search-in-workspace": "next",
+        "@theia/terminal": "next",
+        "@theia/textmate-grammars": "next"
+    },
+    "devDependencies": {
+        "@theia/cli": "next",
+        "@theia/debug": "next"
+    },
```

Comment:
Do you mind rebasing against `master`, and including the following `resolutions` as in other images:
https://github.com/theia-ide/theia-apps/blob/b70f8cf84830b7724acdf5f5315edef6b5b675ed/theia-cpp-docker/latest.package.json#L63-L67

Reply:
FYI, these are temporary work-arounds for a couple of issues we have with Theia platform dependencies. When they're resolved, we'll eventually remove this `resolutions` block.